features/guidance/rotary.feature

```diff
@@ -90,19 +90,19 @@ Feature: Rotary
             | bcegb  | roundabout |
 
        When I route I should get
-           | waypoints | route       | turns                      |
-           | b,d       | bcegb,cd,cd | depart,bcegb-exit-1,arrive |
-           | b,f       | bcegb,ef,ef | depart,bcegb-exit-2,arrive |
-           | b,h       | bcegb,gh,gh | depart,bcegb-exit-3,arrive |
-           | c,f       | bcegb,ef,ef | depart,bcegb-exit-1,arrive |
-           | c,h       | bcegb,gh,gh | depart,bcegb-exit-2,arrive |
-           | c,a       | bcegb,ab,ab | depart,bcegb-exit-3,arrive |
-           | e,h       | bcegb,gh,gh | depart,bcegb-exit-1,arrive |
-           | e,a       | bcegb,ab,ab | depart,bcegb-exit-2,arrive |
-           | e,d       | bcegb,cd,cd | depart,bcegb-exit-3,arrive |
-           | g,a       | bcegb,ab,ab | depart,bcegb-exit-1,arrive |
-           | g,d       | bcegb,cd,cd | depart,bcegb-exit-2,arrive |
-           | g,f       | bcegb,ef,ef | depart,bcegb-exit-3,arrive |
+           | waypoints | route       | turns                           |
+           | b,d       | bcegb,cd,cd | depart,exit rotary right,arrive |
+           | b,f       | bcegb,ef,ef | depart,exit rotary right,arrive |
+           | b,h       | bcegb,gh,gh | depart,exit rotary right,arrive |
```

Comment:
@oxidase this is a special case about us only exiting the roundabout. The `exit nr` is part of the actual instruction. The cucumber case simply does not repeat the exit, which is usually announced at the `enter` instruction.

Do you think we would benefit a lot, from exposing `exit-nr` in the `exit-instruction` as well?